### PR TITLE
Add new coins

### LINKED ISSUE #114: Bitcoin Gold (BTG)
Bitcoin Gold (BTG)

* link to logo: https://btcgpu.org/wp-content/uploads/2017/10/BitcoinGold-logo0k.png
* official website: https://bitcoingold.org

* logo color (HEX): `#eba809`
* bitcointalk.org thread: https://bitcointalk.org/index.php?topic=2171754

### PR DESCRIPTION
**Pull request by edkf**
Hi, thanks for your project. It helps me a lot.
I'm working in a cryptocurrency project, so I add this icons that I didn't find in your webfont.

- Bitcoin Gold BTG (closes #114)
- Bitconnect BCC
- TRON TRX
- Hshare HSR
- ICON ICX
- RaiBlocks XRB
- Populous PPT
- Veritaseum VERI
- vechain VET
- Status SNT
- Binance coin BNB

Thanks a lot.

### CODE REVIEW

File: SVG/PPT.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 714.4 1578"><script id="globalVarsDetection"/><circle cx="294.4" cy="171.1" r="168.3"/><path d="M137.9 367.3H33.3c-17.2 0-31.2 14-31.2 31.2v673.6c0 17.2 14 31.2 31.2 31.2h104.6c7.8 0 14.2-6.3 14.2-14.2V381.5c0-7.9-6.4-14.2-14.2-14.2zM508.1 373.3c-210 0-318-2-318-2v1204h230v-476h106s186 26 186-184v-420s16-122-204-122zm-20 581.5c0 23.5-19 42.5-42.5 42.5h-27.5v-530h27.5c23.5 0 42.5 19 42.5 42.5v445z"/></svg>
```
Comment (epoberezhny) [inline SVG/PPT.svg]: unnecessary id in svg tag
unnecessary script tag

File: SVG/SNT.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 135.3 141.3"><style>.st0{fill:#070400}</style><path class="st0" d="M124.3 25.8c-3-9.8-17.2-17.2-39.8-14.8-19 2-41.2 22.8-45.7 36.8C37.4 52 34.3 65 34.3 65s17.9-2.7 36.1.7 23.4 1.9 30.9.3c11.5-2.3 32.6-9.3 23-40.2zM11 115.5c3.1 9.8 17.2 17.2 39.8 14.7 19-2 41.2-22.8 45.7-36.8 1.4-4.2 4.5-17.2 4.5-17.2s-17.9 2.7-36.1-.7c-18.2-3.4-23.4-1.9-30.9-.3-11.4 2.4-32.6 9.4-23 40.3z"/></svg>
```
Comment (epoberezhny) [inline SVG/SNT.svg]: unnecessary id in svg tag
styles can be inline

File: SVG/trx.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...2 5.7 4.5 11.9 6.6 18 3.7-34.4 7-68.9 10.7-102.5 8.6-81.1 17.6-164.8 25-247.5l.4-2.9s6.6-56.6 9.4-81.6c2-18 4.5-36.5 5.7-54.9v-.4c3.3-29.1 6.1-57.8 9-86.5 4.5-41 8.6-83.6 13.5-125.8-15.1-16.7-29.1-34.3-43-51.1zM1802.9 804.9c-4.5 5.3-9.4 9.8-13.9 14.3-6.6 6.6-12.7 12.7-17.6 19.3l-1.2 1.2C1712 902 1652.6 965.5 1595.6 1027c-45.5 48.8-91.4 98-136.9 146.7 78.3-40.6 156.1-81.6 232.4-121.3 87.3-45.9 177.5-93 267.6-138.9-22.1-15.2-43.9-30.3-65.2-45.5-29.6-20.9-60.3-42.2-90.6-63.1z"/><path d="M1250 0C559.6 0 0 559.6 0 1250s559.6 1250 1250 1250 1250-559.6 1250-1250S1940.4 0 1250 0zm843.4 939.3v4.9l-2.9 4.1c-54.5 78.3-109.4 158.6-162.7 236.5-32.4 47.1-64.8 94.7-97.1 141.8-79.1 115.2-158.2 230.7-237.3 345.9l-324.6 473.4c-7.8 13.9-23.4 22.5-41 22.5h-4.1c-18.9-1.6-34-12.3-39.3-28.7-86.5-241-175-486.5-260.2-723.4C813.5 1108.6 698.8 790.6 586.9 477l-.8-2.9v-18.9l1.2-3.3c2-5.3 6.1-14.8 13.9-22.1l4.9-4.1c4.5-2.9 9.4-4.9 15.2-5.7 19.7-2.5 38.1 2.5 54.1 7 4.9 1.2 9.4 2.5 13.9 3.7 177 43 353.7 86.1 530.7 129.1 195.1 47.5 396.7 96.7 595.1 144.7l.8.4c11.5 3.7 20.9 10.2 29.1 16.4 2 1.2 3.7 2.5 5.3 3.7 74.2 52.5 150 105.3 220.9 154.9 23.9 15.6 23.5 45.1 22.2 59.4z"/><script id="globalVarsDetection"/></svg>
```
Comment (epoberezhny) [inline SVG/trx.svg]: unnecessary id in svg tag
unnecessary script tag in the end

**Comment by AllienWorks** [comment]
Closing, changes already merged in #130

**Pull request closed without merging**